thorlcr/thorutil/thmem.cpp

```diff
@@ -1444,6 +1444,7 @@ rowidx_t CThorSpillableRowArray::save(IFile &iFile, unsigned _spillCompInfo, boo
             ++i;
         }
         writer->flush(NULL);
+        iFileOwner.noteSize(writer->getStatistic(StSizeDiskWrite));
```

Comment:
not directly related to this, but bytesWriten = writer->getPosition (which is only used for tracing) below, could use this same value - and would be good if it did to avoid confusion.
I think move this/change it to offset_t bytesWritten = writer->getStatistic(StSizeDiskWrite); below and call iFileOwner.nodeSize(bytesWritten) there too.